applications/asr_to_llm/metadata.json

```diff
@@ -22,12 +22,7 @@
 			"x86_64",
             "aarch64"
 		],
-		"tags": [
-			"Speech-to-text",
-			"Large Language Model",
-            "ASR",
-            "Local-LLM"
-		],
+		"tags": ["healthcare AI", "audio", "streaming transcription", "transcription summarization", "automatic speech recognition", "local LLM", "streaming", "radiology imaging", "lifesciences"],
```

Comment:
Is "Healthcare AI" a category? can we find a more generic category for this? Maybe "Language Model" ?
"lifesciences" is too generic IMHO